src/fu-engine.c

```diff
@@ -6236,6 +6236,26 @@ fu_engine_ensure_security_attrs(FuEngine *self)
 			}
 			fwupd_security_attr_set_name(attr, name_tmp);
 		}
+		if (fwupd_security_attr_get_title(attr) == NULL)
+			fwupd_security_attr_set_title(attr, fu_security_attr_get_title(attr));
+		if (fwupd_security_attr_get_description(attr) == NULL) {
+			g_autofree gchar *tmp = fu_security_attr_get_description(attr);
+#if 0
```

Comment:
why comment this section out?  seems sensible to me

Reply:
Ahh, because it breaks the gettext trick; changing the daemon locale at runtime just to return the correct translations seems kinda risky. I've removed this and added some notes on the client-side replacement action.